samcli/lib/init/__init__.py

```diff
@@ -28,6 +29,7 @@ def generate_project(
     name=None,
     no_input=False,
     extra_context=None,
+    tracing="Disable",
```

Comment:
My understanding is it's either "enabled" or "disabled". Why not use boolean? 
Also, it looks inconsistent that the "Disable" value is with uppercase "D", but the "enable" value is with lowercase "e".